packages/documentation-site/patternfly-docs/content/get-started/community/community.md

```diff
@@ -70,26 +70,26 @@ import CalendarIcon from '@patternfly/react-icons/dist/esm/icons/calendar-alt-ic
   <p>PatternFly Kotlin is a Kotlin implementation of PatternFly based on fritz2, targeting Kotlin/JS. The goal of this project is to provide all PatternFly components in Kotlin, matching the reactive nature of fritz2. The components use stores, handlers, and other elements from the fritz2 API. For a quick overview, check out the <a href="https://patternfly-kotlin.github.io/patternfly-kotlin-showcase/#home">PatternFly Kotlin showcase</a>.</p>
 
-  <Title size="xl" className="pf-v5-u-mb-sm ws-page-title pf-v5-u-mt-xl" headingLevel="h3"><a href="https://www.sketch.com/s/6ccbd710-267d-4b69-9dae-bc19e1551056">Ansible Component Guide and Sketch Library</a></Title>
+  <Title size="xl" className="pf-v6-u-mb-sm ws-page-title pf-v6-u-mt-xl" headingLevel="h3"><a href="https://www.sketch.com/s/6ccbd710-267d-4b69-9dae-bc19e1551056">Ansible Component Guide and Sketch Library</a></Title>
   <p>The Ansible Component Guide and Sketch Library is an additional resource that designers can use that is built on top of existing PatternFly components. It is a rapid mockup prototyping tool that can be used to quickly put together repeatable design patterns and layouts across projects. Although this is specific to Ansible, many of the components are generalized and can fit many product use cases and are easily detachable to update as needed.</p>
 
-  <Title size="xl" className="pf-v5-u-mb-sm ws-page-title pf-v5-u-mt-xl" headingLevel="h3"><a href="https://github.com/patternfly-yew/patternfly-yew">PatternFly for Yew</a></Title>
+  <Title size="xl" className="pf-v6-u-mb-sm ws-page-title pf-v6-u-mt-xl" headingLevel="h3"><a href="https://github.com/patternfly-yew/patternfly-yew">PatternFly for Yew</a></Title>
   <p>PatternFly for Yew provides PatternFly components for the Yew framework. Yew is a Rust based framework for creating web applications, and is also able to run in the browser using WASM. The project aims to create Yew components for all components and concepts found in PatternFly. For a quick demo and starter template, see the <a href="https://github.com/patternfly-yew/patternfly-yew-quickstart">PatternFly Yew Quickstart.</a></p>
 
-  <Title size="4xl" className="pf-v5-u-mb-lg ws-page-title ws-keep-growing pf-v5-u-mt-3xl" headingLevel="h2">Let's keep growing</Title>
+  <Title size="4xl" className="pf-v6-u-mb-lg ws-page-title ws-keep-growing pf-v6-u-mt-3xl" headingLevel="h2">Let's keep growing</Title>
   <p>The PatternFly community is never finished growing, and we want to keep it that way. Feel free to reach out whenever — we're always open.</p>
 
-  <Grid sm={12} md={6} gutter="sm" className="pf-v5-u-my-lg pf-v5-l-grid pf-m-all-12-col-on-sm pf-m-all-6-col-on-md pf-m-gutter" style={{ maxWidth: '450px' }}>
+  <Grid sm={12} md={6} gutter="sm" className="pf-v6-u-my-lg pf-v6-l-grid pf-m-all-12-col-on-sm pf-m-all-6-col-on-md pf-m-gutter" style={{ maxWidth: '450px' }}>
```

Comment:
I'd expect we have `<Grid>` props that can replace the use of these `.pf-v6-l-grid/.pf-m-*` classes?

Reply:
Followup https://github.com/patternfly/patternfly-org/issues/4021